exercises/pascals-triangle/src/example.c

```diff
@@ -0,0 +1,40 @@
+#include "pascals_triangle.h"
+
+void pt_free(size_t ** mem_block, size_t n)
+{
+   for (size_t i = 0; i < n; i++) {
+      free(mem_block[i]);
+   }
+   free(mem_block);
+}
+
+size_t **pascals_triangle(char *count)
+{
+   if (count == NULL) {
+      return (size_t **) NULL;
+   }
+   size_t **zero = (size_t **) calloc(1, sizeof(size_t *));
+   zero[0] = (size_t *) calloc(1, sizeof(size_t));
+   int i, j;
```

Comment:
and so comparing int and size_t was a bad idea , using int is also good sometimes , because for size_t it has to set widths so it drops the performance... , but that does not matter here.

`src/pascals_triangle.c: In function 'pascals_triangle':
src/pascals_triangle.c:28:25: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
    for (size_t i = 0; i < rows; i++) {
                         ^
src/pascals_triangle.c:32:25: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
    for (size_t i = 1; i < rows; ++i) {
`